src/components/diff/PreviewMainAreaWidget.ts

```diff
@@ -0,0 +1,46 @@
+import { MainAreaWidget } from '@jupyterlab/apputils/lib/mainareawidget';
+import { Message } from '@lumino/messaging';
+import { Widget } from '@lumino/widgets';
+
+export class PreviewMainAreaWidget<
+  T extends Widget = Widget
+> extends MainAreaWidget {
+  /**
+   * Handle on the preview widget
+   */
+  static previewWidget: PreviewMainAreaWidget | null = null;
```

Comment:
Let's make this protected

```suggestion
  protected static previewWidget: PreviewMainAreaWidget | null = null;
```